snoop/management/commands/export-pgp.py

```diff
@@ -0,0 +1,84 @@
+from django.core.management.base import BaseCommand
```

Comment:
Can we name the command `exportpgp.py`? That's the django style, just concatenate words, without a separator,